python/pyspark/pandas/__init__.py

```diff
@@ -144,7 +144,7 @@ def _auto_patch_spark():
 
         from pyspark.sql import dataframe as df
 
-        df.DataFrame.to_koalas = DataFrame.to_koalas
```

Comment:
@itholic can we keep `to_koalas` for backward compatibility for now? We'll have to show a deprecated warning as well.

Reply:
Sounds good. Just created SPARK-35539 and working on it.